osu.Game/Beatmaps/Drawables/DifficultyIcon.cs

```diff
@@ -58,11 +61,14 @@ public virtual Bindable<StarDifficulty> Current
         /// Creates a new <see cref="DifficultyIcon"/>. Will use provided beatmap's <see cref="BeatmapInfo.StarRating"/> for initial value.
         /// </summary>
         /// <param name="beatmap">The beatmap to be displayed in the tooltip, and to be used for the initial star rating value.</param>
+        /// <param name="mods">The mods type beat</param>
```

Comment:
please don't put memes in xmldoc thanks

Reply:
Completely missed this, sorry about that!